README.md

```diff
@@ -1,62 +1,63 @@
+
+The following recommendations apply only to customers who load mParticle via snippet. Customers who self-host via npm can ignore step 1 and jump straight to step 3 after updating their mParticle Braze kit.  Note that this is only one example.  Everywhere you manually call `appboy` needs to be updated similar to the below, and please refer to the Braze Changelog and migration guide as referenced earlier:
+
+* Step 1: Legacy code sample. Find all the places where your code references the `appboy.display` namespace.  Braze has removed all instances of the `display` namespace:
 ```javascript
-let banner = new appboy.ab.Banner(‘test’);
+window.appboy.display.destroyFeed();
 ```
 
-Step 2: Rollout code sample to be used before June 14, 2022:
+Step 2: Roll out code changes to be used before February 15, 2023:
 ```javascript
-let banner;
-if (appboy.hasOwnProperty(‘ab’)) {
-    banner = new appboy.ab.Banner(‘test’);
-} else {
-    banner = new appboy.Banner(‘test’);
+	if (window.appboy) {
+window.appboy.display.destroyFeed();
+} else if (window.braze) {
+	window.braze.destroyFeed();
 }
 ```
-Step 3: After June 14, 2022, you can simplify your code to the following once the mParticle Braze Web kit has been released to our CDN:
+Step 3: After February 1, 2023, you can simplify your code after the mParticle Braze Web kit, which includes Braze SDK V4, has been released to our CDN:
 ```javascript
-let banner = new appboy.Banner(‘test’);
+window.braze.destroyFeed();
 ```
 
-**Push Notifications via service-worker.js**
-
-Additionally, if you are using Push Notifications, we have updated the “service-worker.js” file.  In our testing, Braze’s push notifications work as expected regardless of what version of the service-worker is used, but we recommend updating this to ensure future compatibility.  In your `service-worker.js` file, update the code to reference `https://static.mparticle.com/sdk/js/braze/service-worker-3.5.0.js` instead of `https://static.mparticle.com/sdk/js/braze/service-worker.js`, ie:
+Step 4: Push Notifications via service-worker.js
+If you use Push Notifications, we have updated the `service-worker.js` file.  In our testing, Braze’s push notifications work as expected regardless of what version of the service-worker is used, but we recommend updating this file to ensure future compatibility.  In your `service-worker.js` file, update the code to reference `https://static.mparticle.com/sdk/js/braze/service-worker-4.2.0.js` instead of `https://static.mparticle.com/sdk/js/braze/service-worker-3.5.0.js`.  Your `service-worker.js` file should now contain:
 
 ```javascript
-self.importScripts('https://static.mparticle.com/sdk/js/braze/service-worker-3.5.0.js');
+self.imports('https://static.mparticle.com/sdk/js/braze/service-worker-4.2.0.js')
 ```
 
-# License
```

Comment:
What is the purpose of removing the License? Is this accidental?

Reply:
Accidental. Adding back shortly